### PR TITLE
Detects high count download from a sensitive SAP Privileged account.

### PR DESCRIPTION
**Pull request by gitj121**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    dataTypes:
+      - SAPAuditLog
+tactics:
+  - InitialAccess
+  - Exfiltration
+relevantTechniques:
+  - T1078
+  - T1030
+query: |
+    let priv_users = ('_GetWatchlist("VIPUsers")') ;
```
Comment (devikamehra) [inline Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml:19]: priv_users should be list of distinct users. Fetch results from VIPUsers watchlist and perform distinct operation on User Principal Name.
Schema: https://learn.microsoft.com/en-us/azure/sentinel/watchlist-schemas

File: Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    | join kind=inner 
+    (
+    SAPAuditLog 
+    | where MessageID == "AUY" //AUY= Download bytes
+    | extend ByteCount= toint(replace_string(replace_string(Variable1, ".",""), ",","")), Code=Variable2, Path= Variable3
+    | summarize DownloadsByUser = count(), Paths= make_set(Variable3, 10), ByteCount=sum(ByteCount) by SystemID, ClientID, User, TerminalIPv6, Email, Host, TransactionCode, Instance
+    | where ByteCount > 5000
+    ) on Email, User
+    | project  User, Host, ClientID, Email, MessageID, LogonTypes, SystemID, ByteCount, DownloadsByUser, TerminalIPv6, TransactionCode
+    | extend  AccountCustomEntity = User, HostCustomEntity = Host
```
Comment (devikamehra) [inline Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml:34]: Entity mapping works differently in Hunting queries. For each entity to be mapped, use format like 
| extend Account_0_Name = User in separate line.

File: Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: HostCustomEntity
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: TerminalIPv6
+version: 1.0.0
+kind: Scheduled
```
Comment (devikamehra) [inline Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml:52]: Kind operator is not required for hunting query.

**Review by devikamehra — CHANGES_REQUESTED** [review]

File: Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml (hunk 3 above)
Comment (gitj121) [reply Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml]: Thanks for the review. Added with the requested changes.

**Comment by devikamehra** [comment]
@v-atulyadav The changes are looking good to me. Please check the deployment before merging.

**Review by devikamehra — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**